### ISSUE TITLE
Recommendation System Proof of Concept

### ISSUE THREAD

**Issue by Leodau** [awaketogether/AI#9]
**As a:** Awake member

**I want to:** Have a first approach of the Recommendation Systems

**Description:**
A large part of our project is based on AI and we don't have enought knowledge at the moment to start building a project using neural networks.
The goal of this POC is to get knowledge that we may use in the next sprint.

**Definition of done:**
- [x] Create a Recommendation Systems
- [x] Develop a matrix factorization model
- [x] Develop a softmax model
- [x] Create a report

**Estimated workload:**  3 M/D
